app/models/restaurant.rb

```diff
@@ -0,0 +1,19 @@
+class Restaurant < ApplicationRecord
+  validates :name,
+          presence: true,
+          length: { maximum: 20, minimum: 2, message: "Name length error" }
+
+  validates :rating,
```

Comment:
Rating should not be property of restaurant, it should be a property of the review.
Additionally, the rating of a restaurant should be the average rating of its reviews. Where did you perform this calculation?